presentation/src/main/java/com/architect/coders/mu8/splash/SplashViewModel.kt

```diff
@@ -0,0 +1,48 @@
+package com.architect.coders.mu8.splash
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.architect.coders.mu8.BuildConfig
+import com.architect.coders.mu8.utils.Scope
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+
+class SplashViewModel : ViewModel(), Scope by Scope.Impl() {
+
+    companion object {
```

Comment:
Companion object should go at the end of the class.

Reply:
move companion object end of the class